src/pages/public_registration/PublicPublicationInstance.tsx

```diff
@@ -16,13 +16,24 @@ export const PublicPublicationInstanceJournal: FC<{ publicationInstance: Journal
   const { t } = useTranslation('registration');
   const { type, articleNumber, issue, pages, volume, corrigendumFor } = publicationInstance;
 
+  const fieldTexts = [];
+  if (volume) {
+    fieldTexts.push(`${t('references.volume')} ${volume}`);
+  }
+  if (issue) {
+    fieldTexts.push(`${t('references.issue')} ${issue}`);
+  }
+  if (pages.begin || pages.end) {
```

Comment:
det er vel noe som heter `pages.pages` også?

Reply:
ja, men ikke for `PublicPublicationInstanceJournal` som er denne.
Når det er sagt så har vi faktisk glemt det for Book, så fikser det 😬